### PR TITLE
Implement excerpt entity

### PR DESCRIPTION
**Pull request by nnatter**
Builds upon: https://github.com/sulu/SuluContentBundle/pull/9

### CODE REVIEW

File: Resources/config/doctrine/Excerpt.ExcerptDimension.orm.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                <join-columns>
+                    <join-column name="excerptDimension" referenced-column-name="no" on-delete="CASCADE" nullable="false"/>
+                </join-columns>
+                <inverse-join-columns>
+                    <join-column name="categoryId" referenced-column-name="id" on-delete="CASCADE" nullable="false"/>
+                </inverse-join-columns>
+            </join-table>
+        </many-to-many>
+
+        <many-to-many field="tags" target-entity="Sulu\Bundle\TagBundle\Entity\Tag">
```
Comment (alexander-schranz) [inline Resources/config/doctrine/Excerpt.ExcerptDimension.orm.xml]: @nnatter should this not be TagInterface?

File: Resources/config/doctrine/Excerpt.ExcerptView.orm.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <embeddable name="Sulu\Bundle\ContentBundle\Model\Excerpt\ExcerptView">
+        <field name="title" type="string" nullable="true"/>
+        <field name="more" type="string" nullable="true"/>
+        <field name="description" type="string" nullable="true"/>
+
+        <!-- TODO: how should we handle these relationships?-->
+        <!--
+        <many-to-many field="categories" target-entity="Sulu\Bundle\CategoryBundle\Entity\CategoryInterface">
```
Comment (nnatter) [inline Resources/config/doctrine/Excerpt.ExcerptView.orm.xml]: unfortunately, an embeddable cannot include associations

File: Model/Excerpt/MessageHandler/ModifyExcerptMessageHandler.php [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    ): void {
+        $localizedDraftExcerpt->setTitle($message->getTitle());
+        $localizedDraftExcerpt->setMore($message->getMore());
+        $localizedDraftExcerpt->setDescription($message->getDescription());
+
+        $localizedDraftExcerpt->clearCategories();
+        foreach ($message->getCategoryIds() as $categoryId) {
+            $category = $this->categoryRepository->findCategoryById($categoryId);
+            if (!$category) {
+                throw EntityNotFoundException::fromClassNameAndIdentifier(
```
Comment (nnatter) [inline Model/Excerpt/MessageHandler/ModifyExcerptMessageHandler.php]: should we create explicit exceptions for this?

**Comment by wachterjohannes** [comment]
@nnatter can you rebase this pull-request

**Pull request merged**